cassandra-mesos-model/src/main/proto/io/mesosphere/mesos/frameworks/cassandra/model.proto

```diff
@@ -742,9 +748,13 @@ message NodeJobTask {
      */
     required ClusterJobType jobType = 1;
     /**
-     * Backup name
+     * Backup directory
+     */
+    optional string backupDir = 3;
+    /**
+     * Determine if this is the first node participating in job
      */
-    optional string backupName = 2;
+    optional bool firstNodeInJob = 4;
```

Comment:
@BenWhitehead , 

personally I think, that adding additional type of job is overdesign here.
It will increase the complexity of the implementation without adding
any additional benefit.

Current design includes:
- 2 types of jobs: backup, restore;
- 2 props: backupDir, firstNodeInJob;

Your proposal includes:
- 3 types of jobs: backup, restore, truncate;
- 2 props: backupDir, restoreDir;

Adding a new type of job requires additional control logic
added to 3+ places in code, testing, new review, etc. 

Please take into account, that it will potentially increase time to deliver 
this function in ~3 days (1d for making change, 1d for review and 0.5d for making
review changes), without adding any functional benefit. 

The decision is up to you. Please write, if you prefer design with 3
jobs types and I will implement it. Or may be it would be more practical to
stay with the current design.